### ISSUE TITLE
How do they count? 

### ISSUE THREAD

**Issue by 00lex** [CSSEGISandData/COVID-19#1057]
Hi! Hope you're well. 

Example: 

> 10 confirmed, 0 recovered, 0 death

what if there are no new infections, one dies and another one recover? 

> A1: 10 confirmed, 1 recovered, 1 death: no change on confirmed 
> 
> A2: 9 confirmed, 1 recovered, 1 death
> 
> A3: 8 confirmed, 1 recovered, 1 death 

if A1 is correct, what is when the recovered person gets sick again? 
> B1: 11 confirmed, 1 recovered, 1 death
or
> B2: 11 confirmed, 0 recovered, 1 death
or
> B3: 10 confirmed, 0 recovered, 1 death

**Comment by ballcoach12** [comment]
I would expect the Confirmed count to never decrement. That's the way that I'm viewing it in the model that I've built from the data set.

**Comment by 00lex** [comment]
okay and whats with the reinfected? are they new infections or will they be ignored from count?
doesn't exist any data to reinfected? 
if they are just new infections, we don't know how advanced the infection of a country is.

**Comment by akilelkamel** [comment]
Normally A1 is correct and if the recovered person gets sick again B1 is correct.
The number of confirmed cases increases by the new confirmed cases discovered daily. The deaths and recovered cases increase by the same way.

**Comment by 00lex** [comment]
@akilelkamel 
that's what I thought first, but then we don't have clean data to estimate how many people are left / potential can get sick

**Comment by 00lex** [comment]
..if 50/100 are in the confirmed count and 10 of the 50 are reinfections, 60% of a country could be the next. 
but if we just look at the confirmed count its easy to say its half time. mehh.. :(

**Comment by JiPiBi** [comment]
@akilelkamel  if I have to vote , I prefer B3  because  I think we count a number of people

**Comment by Sleekery** [comment]
All numbers are cumulative. There are a small number of cases where values go down. I'm not sure if that's due to errors on the national reporting or by the data owners.

**Comment by HiIamJeff** [comment]
Metadata is important and hopes this discussion gets more attention! 
IMO, I think it is A1 (As for 2nd question, that would be a lot much variation among countries). Maybe they should include more explanations and assumptions in the field description.

**Comment by RoStall** [comment]
> All numbers are cumulative. There are a small number of cases where values go down. I'm not sure if that's due to errors on the national reporting or by the data owners.

Right, so if I look at Kentucky's time series data, for example, the number recovered goes to 1 at march 16, stays at 1 for march 17, and then goes to zero after until moment of this post -- is this time series data the recoveries of that day, or are they cumulative (i.e., total recoveries as of date?)
